src/cloudCli.ts

```diff
@@ -2,48 +2,53 @@ import * as vscode from 'vscode';
 import { spawn as cpSpawn } from "child_process";
 import { execFile } from './utils';
 import { logger } from './extension';
+import { DbosCloudApp, DbosCloudDatabase } from './dbosCloudApi';
 
-export interface DbosCloudApp {
-    Name: string;
-    ID: string;
-    PostgresInstanceName: string;
-    ApplicationDatabaseName: string;
-    Status: string;
-    Version: string;
-    AppURL: string;
-}
 
-export interface DbosCloudDatabase {
-    PostgresInstanceName: string;
-    HostName: string;
-    Status: string;
-    Port: number;
-    AdminUsername: string;
-}
 
-export async function dbos_cloud_app_status(folder: vscode.WorkspaceFolder) {
+export async function dbosCloudAppStatus(folder: vscode.WorkspaceFolder) {
     const { stdout } = await execFile("npx", ["dbos-cloud", "application", "status", "--json"], {
         cwd: folder.uri.fsPath,
     });
-    const json = JSON.parse(stdout) as DbosCloudApp;
-    logger.debug("dbos_cloud_app_status", { folder: folder.uri.fsPath, stdout: json });
-    return json;
+    const response = JSON.parse(stdout) as DbosCloudApp;
+    logger.debug("dbosCloudAppStatus", { folder: folder.uri.fsPath, response });
+    return response;
+}
+
+export async function dbosCloudAppList(folder: vscode.WorkspaceFolder) {
+    const { stdout } = await execFile("npx", ["dbos-cloud", "application", "list", "--json"], {
```

Comment:
Do we still need to use `dbos-cloud` CLI, or can we directly use `listApps` function in `dbosCloudApi.ts`?

Reply:
Eventually, this code that invokes `npx dbos-cloud` will go away, but that will be a future PR

Reply:
tracking removing dbos-cloud cli integration with #24